Gems/AtomLyIntegration/CommonFeatures/Editor/Scripts/Materials/assign_material_to_all_entities.py

```diff
@@ -0,0 +1,63 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+"""
+    Apply a hard-coded material to all meshes in the level
+"""
+
+import os
+
+import azlmbr.asset as asset
+import azlmbr.bus as bus
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+import azlmbr.math as math
+import azlmbr.editor as editor
+import azlmbr.render as render
+
+###########################################################################
+# Main Code Block, runs this script as main (testing)
+# -------------------------------------------------------------------------
+if __name__ == '__main__':
+    """Run this file as main"""
+
+    
+    # Get the material we want to assign
+    material_asset_path = os.path.join("materials", "types", "debug", "debugvertexstreams.azmaterial")
+    material_asset_id = asset.AssetCatalogRequestBus(
+        bus.Broadcast, "GetAssetIdByPath", material_asset_path, math.Uuid(), False)
+    general.log(f"Assigning {material_asset_path} to all mesh and actor components in the level.")
```

Comment:
I think you should be able to open the asset picker from Python so that the material can be selected instead of hard coded

Reply:
or maybe consider passing it in as an argument, and defaulting to what you have here

Reply:
I'm launching this from the Tools->Other→Python Scripts window, which doesn't have a place to put any input arguments as far as I can tell. I took a brief look at using the Tools->Other->Python Console and the regular console following https://www.youtube.com/watch?v=RTl9lLaWWx0, but neither of those seemed to work for me. So without a straightforward way to actually feed that input, I stuck with this for now. Someone can always update it if they start making heavy use of it and know a better way.